plugins/woocommerce/tests/api-core-tests/tests/shipping/shipping-zones.test.js

```diff
@@ -184,8 +170,13 @@ test.describe( 'Shipping zones API tests', () => {
 
 		const putResponseStateOnlyJSON = await putResponseStateOnly.json();
 		expect( putResponseStateOnly.status() ).toEqual( 200 );
-		expect( putResponseStateOnlyJSON).toHaveLength(0);
 
+		// external host starts with default shipping region
```

Comment:
There is a default shipping region on our external test site.  `wp-env` starts with a completely blank slate.